_scss/tutorial/highlight_block.scss

```diff
@@ -22,20 +21,17 @@
 }
 
 @media (min-width: 480px) {
-.steps {
-    .highlight {
-       width: 50%;
-       float: left;
-       max-width: 600px;
-       margin-left: auto;
-       margin-right: auto;
-       margin-bottom: -500px;
-       padding-bottom: 500px;
-    }
-  }
 }
 
 @media (min-width: 768px) {
+  .steps {
+      .highlight {
+         width: 50%;
+         float: left;
+         margin-bottom: -500px;
+         padding-bottom: 500px;
+      }
+    }
 }
```

Comment:
ouch. didn't test on firefox. seems like the `margin-bottom` is causing that problem; no idea why.